setup.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 3.1.0
+current_version = 3.1.1
```

Comment:
you should leave the bump version change out of a PR. See here https://github.com/ginkgobioworks/edge#versioning for how to do versioning. 
```

Edge is versioned semantically. Continuous integration builds are done automatically on all branches through Travis CI, and tagged commits to master are automatically released to PyPI. To release a new version, bump the version number with the appropriate severity of the changes (major, minor, or patch), and push the resulting tagged commits to the GitHub remote repo:

you@localhost:edge$ docker-compose run --rm edge make bump/patch-ext # Or bump/major, or bump/minor
you@localhost:edge$ git push --tags origin master
If you cannot push to master directly, do the same thing on a new branch and submit a pull request.
```

Reply:
Ah, yeah, I followed those directions, but included the resulting commit in this PR itself.

Do you have a preference whether I submit a new PR for that bumpversion change, or simply push to master, assuming I can do that?

Reply:
No need for open a new PR for bumpversion change, just simply push to master after you merge this code.

Reply:
make sure check out latest master on your local git before doing the bumping thing